gradle/libs.versions.toml

```diff
@@ -47,3 +48,6 @@ google-protobuf = { module = "com.google.protobuf:protobuf-java", version.ref =
 apache-commons-lang = { module = "commons-lang:commons-lang", version.ref = "apache-commons-lang" }
 antlr4 = { module = "org.antlr:antlr4", version.ref = "antlr4" }
 antlr4-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlr4" }
+
+[plugins]
+google-protobuf = { id = "com.google.protobuf", version.ref = "google-protobuf-gradle" }
```

Comment:
I guess a few other things can be moved from `[libraries]` to `[plugins]` like `antlr4`, `diffplug-spotless`, `sonar-scanner-gradle`